.github/workflows/pull-request-container.yml

```diff
@@ -8,26 +7,20 @@ jobs:
   containers:
     name: Create containers
     runs-on: ubuntu-latest
-
     if: |
       github.event.label.name == 'container'
-
     environment:
       name: nonprod
-
     env:
       AWS_CI_ROLE: ${{ secrets.AWS_CI_ROLE }}
-
```

Comment:
I'm not a big fan of how `yq` formats this; do you think I should revert the whitespace changes?

Reply:
I like the steps bellow to have a separation between each of them